docs/source/system_reference_guide/s3-inventory.ipynb

```diff
@@ -0,0 +1,2025 @@
+{
```

Comment:
<p>I think this cell can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>